docs/components_page/components/input/colorpicker.py

```diff
@@ -0,0 +1,22 @@
+import dash_html_components as html
```

Comment:
You can delete this as it's not used in the example.

Reply:
Actually, I need to keep now it because I included your suggestion:
`dbc.Label(["Select a ", html.Span("color", id="color")])`